### DISCUSSION TITLE
Lowdefy on Mac M1

### DISCUSSION THREAD

**Discussion by weiserman** [lowdefy/lowdefy#1374] [Ideas and feature requests]
I am new to Lowdefy - but struggled to get the basic installation working because of Apple Silicon (and SQLite3 actually).

Here is what was required on my local M1:

- Install rosetta via terminal `softwareupdate --install-rosetta`
- Create a new .zshrc file `touch .zshrc`
- Install NVM from command line `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash`
- Open terminal in rosetta mode (Right click on terminal app and set checkbox to, Open using Rosetta)
- Add a new node version `nvm install lts/fermium`
- Set some aliases to easily switch between the node versions `nvm alias intel lts/fermium`
- Set this version current version (runs under x86 architecture)  `nvm use intel`
- Install Lowdefy `npm I lowdefy`

I followed these detailed instructions -> [https://gist.github.com/LeZuse/bf838718ff2689c5fc035c5a6825a11c](https://gist.github.com/LeZuse/bf838718ff2689c5fc035c5a6825a11c)

This should make initial onboarding more painless..

Hope it helps others experimenting with Lowdefy on their M1/M2 Macs,
Warren

**Comment by SamTolmay** [comment]
Hi @weiserman 

Just want to mention that most of our team are using Apple Silicon Macs using native ARM Node.js without any issue. Is the problem with SQLite3 specifically?

**Comment by weiserman** [comment]
Thanks for feedback, my error looked similar to this when attempting to install lowdefy via npm.

```
node_modules/sqlite3: Command failed.
Exit code: 1
Command: node-pre-gyp install --fallback-to-build
```

Running latest node version of Node (which I updated via Brew) - when googling I found number of people that had similar issues. 
Apologies for causing any confusion and inferring any issues with Lowdefy.

Cheers, Warren
